nomad/jitsi-meet-backend.hcl

```diff
@@ -186,11 +193,16 @@ job "[JOB_NAME]" {
 
       config {
         image        = "jitsi/prosody:${var.prosody_tag}"
-        ports = ["prosody-http"]
+        ports = ["prosody-http","prosody-client","prosody-s2s"]
```

Comment:
This doesn't expose prosody-s2s publicly, right?

Reply:
Correct, only exposes it to the 10.x network